src/network/network.h

```diff
@@ -243,4 +224,17 @@ struct ProtocolException : public std::exception {
 	const std::string what_;
 };
 
+/**
+ * The priorities for sending data over the network when using a BufferedConnection.
+ * Data with low priority values is send first even when data with high priority values
+ * have been passed to a BufferedConnection first
+ */
+// The values assigned to the entries are arbitrary, only their order is important
+// No "enum class" on purpose since this has to be interpreted as ints (I need a known ordering)
+enum NetPriority : uint8_t {
```

Comment:
Good idea, so we do not loose pings during filetransfers?

Reply:
Indeed. Is not relevant at the moment but will be with my next branch.